src/validation.c

```diff
@@ -322,6 +322,12 @@ validate_netdef_grammar(const NetplanParser* npp, NetplanNetDefinition* nd, yaml
     if (missing_id_count > 0)
         return TRUE;
 
+    /* filepath: needs to be valid (i.e. no duplicated '//' chars) */
+    if (nd->filepath && g_strrstr(nd->filepath, "//"))
```

Comment:
Well, a path with multiple slashes can be perfectly valid on UNIX. Consecutive slashes must be handled as a single one (unless the slash is the last character in the path and the last component is not a directory). Apparently the way double trailing slashes are handled is defined as implementation specific, but I guess Linux just handle it as though it is a single slash.

So, we can't throw an error because the path has multiple slashes, it would break POLA.

See [pathname](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_271) and [pathname resolution](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap04.html#tag_04_13)

Reply:
ACK. I'll revert this validation commit.